packages/export/src/excel.js

```diff
@@ -60,9 +67,7 @@ export default ({
         excelData[1]
       )
       const readStream = await writeXlsxFile(excelData, { columns })
-      for await (const chunk of readStream) {
-        stream.write(chunk)
-      }
+      await writeStreamData(stream, readStream)
```

Comment:
There's not much point to this function. My suggestion above was so that you could avoid the call to `writeXlsxFile` in this function and instead pass that + writing the stream as an argument. That way you could write a unit test that passes a dummy function that doesn't use `writeXlsxFile` and you can test the functionality without actually writing an excel file.

Reply:
I updated the test and code to reflect this.